deploy/examples/create-external-cluster-resources.py

```diff
@@ -662,7 +662,7 @@ def get_cephfs_node_caps_and_entity(self):
             else:
                 entity = "{}-{}-{}".format(entity,
                                            cluster_name, cephfs_filesystem)
-                caps["osd"] = "allow rw tag cephfs data={}".format(
+                caps["osd"] = "allow rw tag cephfs *={}".format(
```

Comment:
@parth-gr why do you need metadata access for the node user? https://github.com/rook/rook/issues/9227#issuecomment-992445700 mentioned it was working.

Reply:
Maybe the PVC mount would be working, but the user may have not tried reading and writing the data,

If we apply `data=anything` it fails the mount to